.travis.yml

```diff
@@ -28,9 +28,11 @@ notifications:
       on_success: change
       on_failure: always
 
-cache: 
-    - apt
-
+cache:
+  apt: true
+  ccache: true
+  directories:
+  - ~/.ccache
```

Comment:
According to https://docs.travis-ci.com/user/caching/#ccache-cache this is already part of `ccache: true`.